mlflow/store/model_registry/sqlalchemy_store.py

```diff
@@ -258,23 +258,20 @@ def compute_next_token(current_size):
             conditions = []
         elif len(parsed_filter) == 1:
             filter_dict = parsed_filter[0]
-            if filter_dict["comparator"] not in \
+            comparator = filter_dict['comparator'].upper()
+            if comparator not in \
                     SearchUtils.VALID_REGISTERED_MODEL_SEARCH_COMPARATORS:
                 raise MlflowException('Search registered models filter expression only '
                                       'supports the equality(=) comparator, case-sensitive'
                                       'partial match (LIKE), and case-insensitive partial '
                                       'match (ILIKE). Input filter string: %s' % filter_string,
                                       error_code=INVALID_PARAMETER_VALUE)
-            if filter_dict["key"] == "name":
-                if filter_dict["comparator"] == "LIKE":
-                    conditions = [SqlRegisteredModel.name.like(filter_dict["value"])]
-                elif filter_dict["comparator"] == "ILIKE":
-                    conditions = [SqlRegisteredModel.name.ilike(filter_dict["value"])]
-                else:
-                    conditions = [SqlRegisteredModel.name == filter_dict["value"]]
+            if comparator == "LIKE":
```

Comment:
oh! do we have constants from `SearchUtils` for each of these operators? Might be a good idea to add those.

Reply:
Added!